examples/complete/main.tf

```diff
@@ -0,0 +1,45 @@
+resource "random_string" "master_username" {
+  length  = 6
+  special = false
+  upper   = false
+  numeric = false
+}
+
+resource "random_password" "master_password" {
+  length  = 16
+  special = false
+  upper   = false
+}
+
+module "kms_key" {
+  source      = "boldlink/kms/aws"
+  description = "kms key for ${local.cluster_name}"
+  alias_name  = "alias/${local.cluster_name}-key-alias"
+  tags = {
+    environment        = "examples"
+    "user::CostCenter" = "terraform-registry"
+  }
+}
+
+module "complete_cluster" {
+  source                         = "./../../"
+  availability_zones             = data.aws_availability_zones.available.names
```

Comment:
Shoudn't the complete example also have additional parameters? Remember the goal of a complete example is to present an example using all the options